diff --git a/pom.xml b/pom.xml
index 65f78ee7d..a3f2b4bde 100644
--- a/pom.xml
+++ b/pom.xml
@@ -496,7 +496,7 @@
         <dependency>
             <groupId>de.upb.cs.swt</groupId>
             <artifactId>heros</artifactId>
-            <version>1.2.1-SNAPSHOT</version>
+            <version>1.2.1</version>
         </dependency>
         <!-- Uploaded to http://soot-build.cs.uni-paderborn.de/nexus/repository/soot-snapshot/
             from the nightly build server -->
diff --git a/src/main/java/soot/dexpler/DexBody.java b/src/main/java/soot/dexpler/DexBody.java
index 713e7e7ca..b3ce336f6 100755
--- a/src/main/java/soot/dexpler/DexBody.java
+++ b/src/main/java/soot/dexpler/DexBody.java
@@ -865,7 +865,9 @@ public class DexBody {
 
     // We pack locals that are not used in overlapping regions. This may
     // again lead to unused locals which we have to remove.
-    LocalPacker.v().transform(jBody);
+
+    PackManager.v().getTransform("jb.lp").apply(jBody);
+    //LocalPacker.v().transform(jBody);
     UnusedLocalEliminator.v().transform(jBody);
     PackManager.v().getTransform("jb.lns").apply(jBody);
 
